packages/BetterPhpDocParser/Printer/PhpDocInfoPrinter.php

```diff
@@ -284,8 +284,8 @@ private function addTokensFromTo(string $output, int $from, int $to, bool $shoul
 
         // skip extra empty lines above if this is the last one
         if ($shouldSkipEmptyLinesAbove &&
-            \str_contains($this->tokens[$from][0], PHP_EOL) &&
-            \str_contains($this->tokens[$from + 1][0], PHP_EOL)
+            \str_contains((string) $this->tokens[$from][0], PHP_EOL) &&
+            \str_contains((string) $this->tokens[$from + 1][0], PHP_EOL)
```

Comment:
it seems got `PHPStan\Type\UnionType`:

```bash
PHPStan\Type\UnionType #92003
   types: array (2)
   |  0 => PHPStan\Type\Constant\ConstantStringType #92004
   |  |  value: '\n'
   |  |  isClassString: false
   |  1 => PHPStan\Type\Constant\ConstantStringType #91999
   |  |  value: '\r\n'
   |  |  isClassString: false
```

The `$this->nodeTypeAnalyzer->isStringyType` seems needs update for it.

Reply:
It seems it is a mixed type per-defined in docblock 

https://github.com/rectorphp/rector-src/blob/a82a18edd366575db60151880ca44f1a8c2a0120/packages/BetterPhpDocParser/Printer/PhpDocInfoPrinter.php#L72-L75